roadmap/1_core-protocol/upcoming/financial-spam-prevention.md

```diff
@@ -0,0 +1,14 @@
+---
+title: NNS spam protection (remove financial incentive)
+links:
+  Forum Link: https://forum.dfinity.org/t/reproposal-spam-prevention-convert-from-system-based-rewards-to-voter-based-rewards/15352/1
+  Proposal: https://dashboard.internetcomputer.org/proposal/80970
+eta:
+is_community: true
+---
+
+This change suggests a conversion from a system-based rewards model to a voter-based reward model. Voting rewards are allocated under the assumption that every neuron voted on everything. 
+
+This change also includes moving the governance weight back to 20 without incentivising spam. We propose this occur as part of this proposal.
+
+A result of this proposal will be a reduction in total minted ICP due to the fact that some voters do not vote on all proposals or follow a voter for all proposals. In a follow-on proposal, the NNS can determine what to do with that “abandoned” ICP. We suggest an NNS treasury but will leave it to a future proposal to finalize that. 
```

Comment:
I would stick to what is agreed upon and not say anything else here. This feature is about exactly what was proposed, and an NNS treasury is a future discussion (and a controversial one), so I think we should not mix things up.

Reply:
Even though the problem is pretty well known, I would maybe also add one line explaining that currently there is a financial incentive for spam because the rewards are always handed out, and we remove the incentive by changing that.

Reply:
Very good points, which are now incorporated.